core/src/main/java/com/nextdoor/bender/config/BenderConfig.java

```diff
@@ -341,23 +340,24 @@ public static BenderConfig load(String resource) {
      * Read config file
      */
     String data;
+    InputStream is = null;
 
-    URL url;
-    if ((url = Class.class.getResource(resource)) != null) {
-    } else if ((url = Class.class.getResource(resource + ".yaml")) != null) {
+    if ((is = BenderConfig.class.getResourceAsStream(resource)) != null) {
```

Comment:
So nitpicking here.. but from a readability perspective, this is a bit rough. You're using the same method to try to read either a URL, or magically discover and read a .yaml or .json file, right?

Can we make this more readable?